plume-models/src/users.rs

```diff
@@ -51,14 +51,44 @@ use {ap_url, Connection, Error, PlumeRocket, Result};
 
 pub type CustomPerson = CustomObject<ApSignature, Person>;
 
+#[derive(Clone, Debug, DbEnum)]
+#[DieselType = "User_role"]
+pub enum Role {
+    Normal,
```

Comment:
perhaps, "Author"?

Reply:
I don't know, this also includes someone who made an account just to comment